commands/utils/params.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/jfrog/froggit-go/vcsutils"
 	coreconfig "github.com/jfrog/jfrog-cli-core/v2/utils/config"
 	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	clientLog "github.com/jfrog/jfrog-client-go/utils/log"
```

Comment:
Looks redundant. Same in coreconfig.
```suggestion
	"github.com/jfrog/jfrog-client-go/utils/log"
```

Reply:
done